go-controller/pkg/ovn/namespace.go

```diff
@@ -30,12 +30,17 @@ const (
 )
 
 func (oc *Controller) syncNamespaces(namespaces []interface{}) {
```

Comment:
good opportunity to add docstring?

Reply:
heh... not really 😜 I already added it to syncWithRetry.

Reply:
@flavio-fernandes: it's up to you :) But I meant, syncNamespaces, not syncwithRetry. :)